src/components/FolderPage/folderpluslink.jsx

```diff
@@ -0,0 +1,56 @@
+import styles from "./folderpluslink.module.css";
+import linkImg from "../../assets/link.svg";
+import { useState } from "react";
+import FolderPlusModal from "./addToFolder";
+
+function FolderPlusLink() {
+  const [showModal, setShowModal] = useState(false);
+  const [inputLink, setInputLink] = useState('');
+
+  const closeModal = () => {
+    setShowModal(false);
+  };
+
+  const openModal = () => {
+    setShowModal(true);
+  };
+
+  const handleInputChange = (e) => {
+    setInputLink(e.target.value);
+  };
+
+  // 입력값이 비어있는지 확인하여 버튼 활성화/비활성화
+  const isInputEmpty = inputLink.trim() === '';
```

Comment:
아래와 같이 작성할 수도 있답니다~! 

```suggestion
  const isInputEmpty = !!inputLink.trim();
```